README_v2.md

```diff
@@ -0,0 +1,176 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+
+# Welcome to the Apache Solr project!
+-----------------------------------
+
+Solr is the popular, blazing fast open source enterprise search platform
+written in Java and using [Apache Lucene](https://lucene.apache.org/).
+
+For a complete description of the Solr project, team composition, source
+code repositories, and other details, please see the Solr web site at
+https://solr.apache.org/
+
+## Online Documentation
+
+This README file only contains basic instructions.  For comprehensive documentation,
+visit the [Solr Reference Guide](https://solr.apache.org/guide/).
+
+## Getting Started
+FIXME maybe put the tutorial that Noble or Ishan wrote???   Then at the end put the examples?
+
+### Solr Examples
+
+Solr includes a few examples to help you get started. To run a specific example, enter:
+
+```
+  bin/solr -e <EXAMPLE> where <EXAMPLE> is one of:
+    FIXME the labels in the README do NOT match what is in the bin/solr start -help, and are better!
+    cloud:         SolrCloud example
+    techproducts:  Comprehensive example illustrating many of Solr's core capabilities
+    schemaless:    Schema-less example (schema is inferred from data during indexing)
+    films:         Example of starting with _default configset and adding explicit fields dynamically    
+```
+
+For instance, if you want to run the SolrCloud example, enter:
+
+```
+  bin/solr -e cloud
+```
+
+### Running Solr in Docker
+
+You can run Solr in Docker via the [official image](https://hub.docker.com/_/solr).
+
+To run Solr in a container and expose the Solr port, run:
+
+`docker run -p 8983:8983 solr`
+
+In order to start Solr in cloud mode, run the following.
+
+`docker run -p 8983:8983 solr solr-fg -c`
+
+For documentation on using the official docker builds, please refer to the [DockerHub page](https://hub.docker.com/_/solr).  
+Up to date documentation for running locally built images of this branch can be found in the [local reference guide](solr/solr-ref-guide/src/running-solr-in-docker.adoc).
+
+There is also a gradle task for building custom Solr images from your local checkout.
```

Comment:
Perhaps add a link to `dev-docs/docker.adoc` - eeh, no, wait that does not exist, I meant `help/docker.txt` -- eh, sorry, wrong again, it should be `solr/docker/gradle-help.txt` (why on earth do we have dev-docs spread out this much?) so we don't need to say so much about building Docker locally in the main README.

Hmm, looking a few lines down, I now see that `./gradlew helpDocker` actually points to that txt file, but that is not consistent either, would expect all help files to exist in help/.. Well, well.

Reply:
Sorry for that placement, there was discussion at the time, but it is long forgotten. I'd be happy to move it to wherever it will make the most sense 🙂